spring-cloud-gcp-examples/spring-cloud-gcp-spring-integration-pubsub-example/README.adoc

```diff
@@ -0,0 +1,19 @@
+= Spring Integration Channel Adapters for Google Cloud Pub/Sub Code Sample
+
+This code sample contains a couple of apps.
+One app gets a message provided by the user through a webpage and publishes it to a Google Cloud
+Pub/Sub topic.
+The other app subscribes to that topic and logs received messages.
+
+== Setup & Configuration
+
+1. Configure your GCP project ID and credentials by following
+link:../../spring-cloud-gcp-starters/spring-cloud-gcp-starter-core[these instructions], or just let
+Spring discover those parameters for you.
+2. Go to the https://pantheon.corp.google.com/cloudpubsub/topicList[Google Cloud Console] and create
```

Comment:
I skipped this step initially, and tried publishing a message anyway, and I saw no error in the logs or the web UI, yet no message went anywhere.

Reply:
That happens because the call is made asynchronously.
Since #166 was merged, you can set the mode to synchronous and the error will show up. We mention in the README and ref doc that on async mode, errors won't be displayed.

Reply:
Actually, for the example, I'll set sync to `true` so errors become visible!